runtimes/core/src/api/call.rs

```diff
@@ -120,12 +120,19 @@ impl ServiceRegistry {
         target: EndpointName,
         data: JSONPayload,
         source: Option<Arc<model::Request>>,
+        opts: Option<Arc<api::CallOpts>>,
```

Comment:
Does it need to be wrapped in an `Arc`?

Reply:
good question, it feels like I added it there for a reason, but I cant remember why, will remove